contracts/identity/lib.rs

```diff
@@ -158,6 +180,11 @@ mod identity {
 			self.identity_of.get(owner)
 		}
 
+		#[ink(message)]
+		pub fn network_name_of(&self, network_id: NetworkId) -> Option<String> {
+			self.network_name.get(network_id)
```

Comment:
Btw, maybe we should rename the storage value `network_name` to `network_name_of`. I think it would be better.

Reply:
If you agree to do this, this small change can probably be done in this PR also.